client/scss/components/_find_us.scss

```diff
@@ -52,12 +55,51 @@
 }
 
 .find-us__map {
-  max-width: 100%;
+  width: 100%;
```

Comment:
Will need to adjust the srcset and sizes attributes, as the image can now appear larger on smaller screens